features/steps/steps.go

```diff
@@ -1,79 +1,76 @@
 package steps
 
 import (
+	"context"
 	"encoding/json"
+	dpMongoDriver "github.com/ONSdigital/dp-mongodb/v2/pkg/mongodb"
```

Comment:
For package names and import handles It's idiomatic to not use mixedCaps, underscores, dashes etc and to keep as simple as possible.. something like `dpmongo` or `mongodriver` would be more appropriate.. But is `mongodb` even unavailable here?

Reply:
package name has been updated to follow current standard